.github/workflows/connector-test-command.yml

```diff
@@ -117,10 +117,7 @@ jobs:
       - name: Fetch connector secrets
         if: matrix.connector
         run: |
-          python -m pip install --upgrade pip
-          python -m pip install pipx
-          uv tool run airbyte-cdk[dev] secrets fetch \
-            --connector-name=${{ matrix.connector }} \
+          airbyte-cdk secrets fetch ${{ matrix.connector }} \
```

Comment:
No each connector type's `install` task (run before this one) will make sure that `airbyte-cdk` is installed. That simplifies this step.